renovate.json

```diff
@@ -0,0 +1,35 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "ignorePaths": [
+    "example-problems/**",
+    "boms/cloud-oss-bom/pom.xml",
+    "boms/convergence-check/pom.xml"
```

Comment:
I think you need to exclude boms/upper-bounds-check/ too.

Reply:
Updated this to ignore "boms/**".